tox.ini

```diff
@@ -125,8 +125,8 @@ multi_line_output = 3
 include_trailing_comma = True
 force_grid_wrap = 0
 line_length = 99
-known_first_party = tox
-known_third_party = apiclient,git,httplib2,oauth2client,packaging,pkg_resources,pluggy,py,pytest,setuptools,six
+known_first_party = tox,tests
+known_third_party = apiclient,git,httplib2,oauth2client,packaging,pkg_resources,pluggy,py,pytest,setuptools,six,tests,toml
```

Comment:
@asottile for some reason tests shows up as third party even though it is marked as first party